### PR TITLE
Refactor parameters

### PR DESCRIPTION
**Pull request by Vahe1994**
1.  deleted folder "build" from lm_evaluation_harness
2. updated default parameters for wandb and percdamp
3. modified Readme.md(added citation, and small changes )
4. changed default behaviour to run leave-one-out evaluation when detecting outliers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The code requires the LLaMA model to be dowloaded in Hugging Face format and sav
 
 ### Perplexity benchmarks:
 This script compresses the model and then tests its performance in terms of perplexity using Wikitext2,
```
Comment (justheuristic) [inline README.md:45]:
```suggestion
This script compresses the model and then tests its performance in terms of perplexity using WikiText2, 
```

File: main.py [hunk 2]
```diff
@@ -369,7 +369,7 @@ def forward(self, inp, **kwargs):
     parser.add_argument("--seed", type=int, default=0, help="Seed for sampling the calibration data.")
     parser.add_argument("--nsamples", type=int, default=128, help="Number of calibration data samples.")
     parser.add_argument(
-        "--percdamp", type=float, default=0.01, help="Percent of the average Hessian diagonal to use for dampening."
+        "--percdamp", type=float, default=1., help="Percent of the average Hessian diagonal to use for dampening."
```
Comment (justheuristic) [inline main.py:372]: [Nit] Can we please keep the default at 0.01? some people already measured the perplexity with 0.01, some of our configs also use this.

If the majority of code owners prefer that we switch, I will relent.

File: spqr_engine.py [hunk 3]
```diff
@@ -63,8 +63,8 @@ def quantize(
         :note: if keep_last_columns > 0, quantized_dequantized_weights[-keep_last_columns:] will be non-quantized
         :param permutation_order: re-order input features using a certain policy
         :param keep_H: if False, delete the accumulated hessian during quantize; if False, keep the accumulated hessian
-        :param fit_quantizer_without_outliers: if True, fit quantizer parameters without unstrucutred outliers
-            the algorithm is a bit complex, please see the code for details
+        :param simplified_outliers: if True,do not perform leave-one-out evaluation when detecting outliers;
+        works faster, but generally worse in perplexity
```
Comment (justheuristic) [inline spqr_engine.py:67]:
```suggestion
            works faster, but generally worse in perplexity
```

**Review by justheuristic — APPROVED** [review]
Great work! I've left a few minor comments. If you agree with those - and if you wish - I can help resolving them as well.

File: main.py (hunk 2 above)
Comment (Vahe1994) [reply main.py:372]: Ok. I will keep 0.01, I think,  it is not critical.
Although, if I remember correctly all our runs where with percdamp = 1.

**Comment by justheuristic** [comment]
l g tm

**Review by justheuristic — APPROVED** [review]

File: main.py [hunk 4]
```diff
@@ -517,8 +529,8 @@ def forward(self, inp, **kwargs):
         if args.new_eval:
             neweval_str = "_new_eval"
         wandb.init(
-            entity="rock-and-roll",
-            project="SpQR-for-Falcon",
+            entity=args.wandb_entity,
+            project=args.wandb_project,
```
[the remark this answers is not in the document]
Comment (Vahe1994) [reply main.py]: Agreed

**Pull request merged**